braindecode/samplers/base.py

```diff
@@ -18,8 +18,8 @@ class RecordingSampler(Sampler):
     Parameters
     ----------
     metadata : pd.DataFrame
-        DataFrame with at least one of {subject, session, run} columns for each
-        window in the BaseConcatDataset to sample examples from. Normally
+        DataFrame with at least one of {subject, session, run, recording} columns
```

Comment:
what is a recording? how does a session differ from a recording?

Reply:
I don't know, sounds like a question for other maintainers.

Reply:
It's not consistent across all datasets; some use recording as a file unit, others use session, and some sessions have many recordings (I don't know why).